cellxgene_schema_cli/tests/test_schema_compliance.py

```diff
@@ -162,6 +162,20 @@ def test_shapes(self, validator_with_adata):
             in validator.errors
         )
 
+    def test_csc_matrix_invalid(self, validator_with_adata):
```

Comment:
while looking at the issue brought up in [this thread](https://czi-sci.slack.com/archives/C08MF1AJ6F5/p1745437756457979), i thought that we weren't raising a validation error at all for unsupported CSC matrices, so i added test coverage to check that. it turns out we already are, but i figured having test coverage for this is a good idea so i kept it in this PR lol